MoneySplitter.Win10/ViewModels/AddTransactionViewModel.cs

```diff
@@ -0,0 +1,218 @@
+        private bool _isLoading;
+        private bool? _isSelfCollabarator;
+        private bool _isErrorVisible;
+        private ErrorDetailsModel _errorDetailsModel;
+        #endregion
+
+        #region Properties
+        public ObservableCollection<UserModel> Friends
+        {
+            get { return _friends; }
+            set
+            {
+                _friends = value;
+                NotifyOfPropertyChange(nameof(Friends));
+            }
+        }
+
+        public ObservableCollection<UserModel> Collabarators
+        {
+            get { return _collabarators; }
+            set
+            {
+                _collabarators = value;
+                NotifyOfPropertyChange(nameof(Collabarators));
+            }
+        }
+
+        public AddTransactionModel AddTransactionModel
+        {
+            get { return _addTransactionModel; }
+            set
+            {
+                _addTransactionModel = value;
+                NotifyOfPropertyChange(nameof(AddTransactionModel));
+            }
+        }
+
+        public string LabelForTransactionImage
+        {
+            get { return _laberlForTransactionImage; }
+            set
+            {
+                _laberlForTransactionImage = value;
+                NotifyOfPropertyChange(nameof(LabelForTransactionImage));
+            }
+        }
+
+        public bool IsNotCollaboratorsTextVisibility
+        {
+            get { return _isNoCollaboratorsTextVisibility; }
+            set
+            {
+                _isNoCollaboratorsTextVisibility = value;
+                NotifyOfPropertyChange(nameof(IsNotCollaboratorsTextVisibility));
+            }
+        }
+
+        public bool IsLoading
+        {
+            get { return _isLoading; }
+            set
+            {
+                _isLoading = value;
+                NotifyOfPropertyChange(nameof(IsLoading));
+            }
+        }
+
+        public bool IsErrorVisible
+        {
+            get { return _isErrorVisible; }
+            set
+            {
+                _isErrorVisible = value;
+                NotifyOfPropertyChange(nameof(IsErrorVisible));
+            }
+        }
+
+        public bool? IsSelfCollabarator
+        {
+            get { return _isSelfCollabarator; }
+            set
+            {
+                _isSelfCollabarator = value;
+                NotifyOfPropertyChange(nameof(IsSelfCollabarator));
+            }
+        }
+
+        public ErrorDetailsModel ErrorDetailsModel
+        {
+            get { return _errorDetailsModel; }
+            set
+            {
+                _errorDetailsModel = value;
+                NotifyOfPropertyChange(nameof(ErrorDetailsModel));
+            }
+        }
+        public string NOT_COLLABARATORS = Defines.AddTransaction.NOT_COLLABARATORS;
+        #endregion
+
+        public AddTransactionViewModel(ITransactionsManager transactionsManager, IFriendsManager friendsManager,
```

Comment:
If you have a lot of parameters, move each of them to separate line
AddTransactionViewModel(
      ITransactionsManager manager,
      IFriendsManager.....

Reply:
separated parameter constructor